src/recordings.ts

```diff
@@ -0,0 +1,65 @@
+import { ZoomMeetingRecordingFileType, ZoomMeetingRecordingType } from './common';
+import request from './util/request';
+
+export type GetAccountRecordingsParams = {
+  page_size?: number;
+  next_page_token?: string;
+  from?: string;
+  to?: string;
+};
+
+export type GetAccountRecordingsResponse = {
+  next_page_token: string;
+  from: string;
+  to: string;
+  page_size: number;
+  total_records: number;
+  meetings: {
```

Comment:
This looks to be identical to the `GetMeetingRecordingsResponse` type in `meetings.ts`, minus the `account_id` field. Can you reconcile them so we can prevent duplicate type definitions as much as possible?

Reply:
Hah, my bad. They are identical, even the `account_id` is present in both